src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,31 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
+    }
+
+    /**
+     * Use it to capture the argument that satisfies the matcher. This method <b>must be used inside of verification</b>.
+     * <p>
```

Comment:
I think users of that nice new method are not that much interested in how it is implemented under the hood. As a user I want to know how to use it  (sample code) and a link to the new ArgumentMatchers-class would be useful.